tests/integration/features/steps/integration_steps.py

```diff
@@ -1114,3 +1390,21 @@ def write_dummy_file(path, mtime_str, contents=None):
     mtime = (t - datetime.datetime(1970, 1, 1)).total_seconds()
     atime = mtime
     os.utime(path, (atime, mtime))
+
+
+def get_mgmt_api_jars(
+        url="https://github.com/datastax/management-api-for-apache-cassandra/releases/download/v0.1.17/jars.zip"):
+
+    zip_file = requests.get(url, stream=True)
+
+    with open("/tmp/mgmt_api_jars.zip", "wb") as mgmt_api_jars:
+        for chunk in zip_file.iter_content(chunk_size=4096):
+            if chunk:
+                mgmt_api_jars.write(chunk)
+
+    with ZipFile('/tmp/mgmt_api_jars.zip', 'r') as zip_ref:
+        list_of_names = zip_ref.namelist()
+        for file_name in list_of_names:
+            if file_name.endswith('.jar'):
+                if 'mgmtapi-agent' in file_name or 'mgmtapi-server' in file_name:
+                    zip_ref.extract(file_name, '/tmp')
```

Comment:
This is just a test utility method to pull down the Management API jars so they can be unzipped and used to embed into the running CCM Cassandra instance for the test. It might be worth it to configure the version to use (it's called on line 334 [above](https://github.com/thelastpickle/cassandra-medusa/pull/237/files#diff-e513e4ce15a41a09a27c952d7e5d7afc4a04ed85d3022707ca912ab26c5cce14R334)). For now I have it defaulted to v0.1.17 here (the current latest).